packages/fiori/src/themes/ShellBar.css

```diff
@@ -52,14 +52,23 @@
 	word-spacing: inherit;
 }
 
+.ui5-shellbar-button[icon-only] {
+	color: var(--sapShell_InteractiveTextColor);
+}
+
+.ui5-shellbar-button[icon-only]:hover:not(:active) {
+	color: var(--sapShell_InteractiveTextColor);
+}
+
+
 ::slotted([ui5-button][slot="startButton"]) {
 	margin-inline-start: 0;
 }
 
-::slotted([ui5-button][slot="startButton"]:hover),
-.ui5-shellbar-menu-button.ui5-shellbar-menu-button--interactive:hover,
-.ui5-shellbar-button:hover,
-.ui5-shellbar-image-button:hover {
+::slotted([ui5-button][slot="startButton"]:hover:not([active])),
```

Comment:
In the rest of the file we have styles which are with stronger rules for the active state. Rendering those 4 new additions irrelevant.

Are you sure they need to be added and if there is any visual difference with or without them?